docs/reference/ingest/processors/inference.asciidoc

```diff
@@ -15,20 +15,28 @@ ingested in the pipeline.
-| Name                                  | Required  | Default                        | Description
-| `model_id` .                          | yes       | -                              | (String) The ID or alias for the trained model, or the ID of the deployment.
-| `input_output`                        | no        | -                              | (List) Input fields for inference and output (destination) fields for the inference results. This options is incompatible with the `target_field` and `field_map` options.
-| `target_field`                        | no        | `ml.inference.<processor_tag>` | (String) Field added to incoming documents to contain results objects.
-| `field_map`                           | no        | If defined the model's default field map | (Object) Maps the document field names to the known field names of the model. This mapping takes precedence over any default mappings provided in the model configuration.
+| Name                                  | Required  | Default                                    | Description
+| `model_id` .                          | yes       | -                                          | (String) The ID or alias for the trained model, or the ID of the deployment.
+| `input_output`                        | no        | -                                          | (List) Input fields for {infer} and output (destination) fields for the {infer} results. This option is incompatible with the `target_field` and `field_map` options.
+| `target_field`                        | no        | `ml.inference.<processor_tag>`             | (String) Field added to incoming documents to contain results objects.
+| `field_map`                           | no        | If defined the model's default field map   | (Object) Maps the document field names to the known field names of the model. This mapping takes precedence over any default mappings provided in the model configuration.
 | `inference_config`                    | no        | The default settings defined in the model  | (Object) Contains the inference type and its options.
-| `ignore_missing`                      | no        | `false`  | (Boolean) If `true` and any of the input fields defined in `input_ouput` are missing then those missing fields are quietly ignored, otherwise a missing field causes a failure. Only applies when using `input_output` configurations to explicitly list the input fields.
+| `ignore_missing`                      | no        | `false`                                    | (Boolean) If `true` and any of the input fields defined in `input_ouput` are missing then those missing fields are quietly ignored, otherwise a missing field causes a failure. Only applies when using `input_output` configurations to explicitly list the input fields.
 include::common-options.asciidoc[]
 |======
 
+IMPORTANT: The usage of `input_output` field and `target_field`, `field_map` 
+fields are incompatible. Either use the `input_output` option or the 
```

Comment:
> Either use the `input_output` option or the `target_field` and `field_map` option.

This makes me wonder whether that choice is mandatory, though in the table they're all listed as optional. 
IMO if the choice is not mandatory (i.e. you can not specify *any* of them), I'd suggest removing this sentence since the first sentence in the paragraph should be sufficient.